lightbluetent/api.py

```diff
@@ -191,11 +223,16 @@ def is_meeting_running(self, params):
         running = False
         response = self.request("isMeetingRunning", params)
 
-        if "returncode" not in response:
+        if response is None:
```

Comment:
Response should always be an XML dict even if failed. This condition might never be satisfied, better to check the error or code of the XML response.

Reply:
When the response isn't actually from BBB (e.g. wrong address, can't be reached), there is no returncode field. Example response:

`<Map><timestamp>1600544194422</timestamp><status>404</status><error>Not Found</error><message>No message available</message><path>/bigbluebutton/apiisMeetingRunning</path></Map>`

Reply:
Yes, I also neglected to read that you had put in that check in the `request` function and returned `None`. So this makes sense.